_includes/layout/page-header.html

```diff
@@ -1,12 +1,19 @@
 <!-- page-header -->
 <section class="page-intro">
-	{% if page.ee_only == true %}
-		<span class="ee-only">{{site.data.var.ee}} only</span>
-	{% endif %}
+{% if page.guide_version != site.version %}
+<div class="message-banner versions">
```

Comment:
Ideally, html should be indented for easier readability of the nested elements.

Reply:
Fixed. Had to get rid of tabs.